content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -175,11 +153,33 @@ metadata:
     dd_k8s_cluster: example-cluster
 ```
 
+## Change duration to wait for resources health
+The duration reported in deployment events matches the sync duration in Argo CD. However, the sync duration generally represents the time spent by Argo CD to sync the Git repository state and the Kubernetes cluster state.
+This means that what happens after the sync (for example, the time spent by the Kubernetes resources to start up), is not included in the duration.
 
-## Visualize deployments in Datadog
+To change the duration reported to wait until the configured resources have started up and reached a healthy state, you can add a [PostSync Hook][19] to the resources that your Argo CD application monitors.
+The PostSync Hook will run after all the resources have reached a Healthy state, and the Argo CD sync will wait on the PostSync Hook result to update the application status as Healthy.
 
-The [**Deployments**][6] and [**Executions**][7] pages populate with data after a deployment is executed. For more information, see [Search and Manage][9] and [CD Visibility Explorer][10].
+Below is represented an example of a PostSync Hook Job that runs a simple `echo` command.
```

Comment:
People have to add a noop container in order to get the hook to be sent after the application is healthy? Is there any other alternative? I feel it's odd to just say that this is an example without explaining what else they could do (or not do)